tests/package.json

```diff
@@ -26,12 +29,13 @@
     "wait-for-expect": "^3.0.2"
   },
   "devDependencies": {
+    "@types/jest": "^29.5.14",
     "@types/node-fetch": "^2.6.6",
-    "jest": "^27.4.0",
+    "babel-jest": "^29.7.0",
+    "jest": "^29.7.0",
     "wait-for-expect": "^3.0.2"
   },
   "engines": {
-    "node": ">=20"
-  },
-  "version": "0.0.0"
+    "node": ">=23"
```

Comment:
So Node v23 is necessary for running the tests ? Should it be mandatory for the backend also ?

Reply:
You mean for semapps? 
I think so but I haven't stepped through (IIRC, it worked for me with node 22 before).
I think the later versions of node 22 support esm requiring by default (and typscript). But not all of them. So I thought, 23 is safer to have...

Reply:
I mean for the ActivtityPods backend. On the next branch we switched to Node 22:
https://github.com/activitypods/activitypods/blob/d4e94b2cbe69d79b05ea053958c61c4bdb3b871a/pod-provider/backend/package.json#L59
So for consistency it would be better to have the same version on the whole repository.
This makes me think we need to merge the next branch... :thinking:

Reply:
I changed the PR to go into next.

I'd suggest that we put PRs on a permanent `dev` or `next` only (unless they are hotfixes) so we keep master more stable and have a more consistent practice.
Or we use `master` only but that can lead to issues when people who want to check out the project are left with a non-working version that they just pulled.

Reply:
@SlyRock wants to make a proposal about the branches.